assets/app/scripts/directives/resources.js

```diff
@@ -45,9 +53,28 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        pods: '='
+        pods: '=',
+        projectName: '@?' //TODO optional for now
       },
-      templateUrl: 'views/_pods.html'
+      templateUrl: 'views/_pods.html',
+      controller: function($scope) {
+        $scope.phases = [
+          "Failed",
+          "Pending",
+          "Running",
+          "Succeeded",
+          "Unknown"
+        ];
+        $scope.expandedPhase = null;
+        $scope.warningsExpanded = false;
+        $scope.expandPhase = function(phase, warningsExpanded, $event) {
+          $scope.expandedPhase = phase;
```

Comment:
If you have multiple deployments with collapsed pods, does this only expand one or all?